src/main/java/com/github/princesslana/slothbot/commands/CounterConfigCommand.java

```diff
@@ -0,0 +1,65 @@
+package com.github.princesslana.slothbot.commands;
+
+import com.github.princesslana.slothbot.Discord;
+import com.github.princesslana.slothbot.MessageCounter;
+import com.github.princesslana.smalld.SmallD;
+import com.google.common.base.Preconditions;
+import disparse.discord.smalld.DiscordRequest;
+import disparse.discord.smalld.DiscordResponse;
+import disparse.parser.dispatch.IncomingScope;
+import disparse.parser.reflection.*;
+
+public class CounterConfigCommand {
+
+  private final SmallD smalld;
+  private final DiscordRequest request;
+  private final MessageCounter counter;
+  private final String guildId;
+
+  public CounterConfigCommand(SmallD smalld, DiscordRequest request, MessageCounter counter) {
+    this.smalld = smalld;
+    this.request = request;
+    this.counter = counter;
+    this.guildId = Discord.getGuildId(request);
+  }
+
+  @CommandHandler(
+      commandName = "excludebotmsgs",
+      aliases = {"ebmsg"},
+      description = "Exclude bot messages for slowmode calculation in current server",
+      acceptFrom = IncomingScope.CHANNEL)
+  @Usages({
+    @Usage(
+        usage = "",
+        description = "Exclude bot messages for slowmode calculation in current server")
+  })
+  public DiscordResponse excludeBotMsgs() {
```

Comment:
Would this be better as a single command?

e.g., `sb!count_bots true` or `sb!count_bots false` ?